concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+Williamson-1973-118-45	45	give: «dash»	donner: faire cadeau	geben: schenken	1447	GIVE
+Williamson-1973-118-46	46	go: walk	aller: marcher	gehen, fahren: zu Fuss gehen	695	GO
+Williamson-1973-118-47	47	grasshopper	sauterelle	Grashüpfer, Heuschrecke	607	GRASSHOPPER
+Williamson-1973-118-48	48	grow: become strong	croître: grandir	wachsen: stark werden	479	GROW
+Williamson-1973-118-49	49	guineafowl	pintade	Perlhuhn	456	GUINEA FOWL
+Williamson-1973-118-50	50	head	tête	Kopf	1256	HEAD
+Williamson-1973-118-51	51	heart	coeur	Herz	1223	HEART
+Williamson-1973-118-52	52	heavy	lourd (être)	schwer	1210	HEAVY
+Williamson-1973-118-53	53	hoe: cultivate	piocher: cultiver	hacken: anbauen	1541	CULTIVATE
+Williamson-1973-118-54	54	kill	tuer	töten	1417	KILL
+Williamson-1973-118-55	55	leg: foot	jambe: pied	Bein: Fuss	2098	FOOT OR LEG
+Williamson-1973-118-56	56	leopard: «panther»	léopard: «panthère»	Leopard: Panther	1139	LEOPARD
+Williamson-1973-118-57	57	lie down: stretch oneself:: sleep	se coucher: s’allonger:: dormir	liegen, sich hinlegen: sich (aus)strecken :: schlafen		
+Williamson-1973-118-58	58	lick	lécher	lecken	319	LICK
+Williamson-1973-118-59	59	man: husband	homme: mari	Mann: Ehemann, Gatte, Gemahl	2646	HUSBAND OR MALE PERSON
+Williamson-1973-118-60	60	meat	viande	Fleisch	634	MEAT
+Williamson-1973-118-61	61	mould	mouler	abformen, modellieren		
+Williamson-1973-118-62	62	mount	monter	hinaufgehen, besteigen, ersteigen	705	GO UP (ASCEND)
+Williamson-1973-118-63	63	name	nom	Name	1405	NAME
+Williamson-1973-118-64	64	night: day of 24 hours	nuit: jour de 24 heures	Nacht: Tag von 24 Stunden		
+Williamson-1973-118-65	65	oil palm: fan palm:: dum palm	palmier à huile: ?:: ?	Ölpalme: ?::?		
+Williamson-1973-118-66	66	person	personne	Person, Mensch	683	PERSON
+Williamson-1973-118-67	67	pig (domestic): (wild)	cochon: phacochère	Ferkel: Wildschwein [?]	1337	PIG
+Williamson-1973-118-68	68	press: squeeze out: milk	presser::traire	drücken, pressen: auspressen, ausdrücken:: melken		
+Williamson-1973-118-69	69	quarrel v. and n.	se quereller, disputer: querelle	(sich) streiten: Streit		
+Williamson-1973-118-70	70	raphia palm (vinifera): (hookeri):: date palm	raphia (vinifera): hookeri:: dattier	?: ?:: Dattelpalme		
+Williamson-1973-118-71	71	refuse: retain	refuser: retenir	verweigern, abschlagen: zurückhalten, einbehalten		
+Williamson-1973-118-72	72	resemble: be equal to	ressembler: être égal à	ähnlich sein: gleichen, gleich sein	1777	RESEMBLE
+Williamson-1973-118-73	73	road: path	route: sentier	(Land)Strasse: Pfad, Weg	2457	PATH OR ROAD
+Williamson-1973-118-74	74	roast on coals	griller au feu de bois	auf Kohlen rösten		
+Williamson-1973-118-75	75	rot	pourrir	verfaulen, verwesen	2907	ROT
+Williamson-1973-118-76	76	rubbish-heap	tas d’ordures	Schutthaufen		
+Williamson-1973-118-77	77	send: work n.	envoyer: travail	schicken, senden: Arbeit:: arbeiten		
+Williamson-1973-118-78	78	set a trap	poser un piège	eine Falle stellen		
```

Comment:
1170?

Reply:
I don't think 1170 would fit here because it refers to the act of catching, not preparing the trap.

Reply:
True, thanks.